src/Velociraptor.js

```diff
@@ -25,6 +25,10 @@ module.exports = React.createClass({
         this._updateChildren(this.props.children);
     },
 
+    componentWillUnmount: function() {
+        this._updateChildren([]);
```

Comment:
surely this doesn't do anything? it just sets `this.children = {}`, which has no effect, because `render` won't be called again? Or am I missing something